### DISCUSSION TITLE
How can I open a local file quickly?

### DISCUSSION THREAD

**Discussion by chipbite** [jgraph/drawio#2803] [Q&A]
Hi! 👋 😊
I use drawio with confluence or local files (that often live in a git repo). 
I use drawio from web since it supports minimal theme, keyboard control of the palettes and has fullscreen support, which affords me best control of the canvas screen real estate - the desktop app lacks this (some or all).

So, to open a local file: I have not found an effective way to do so. 
I have to open the editor, and then click File, click Open From... click Device. Then navigate to the folder and click to select the file.  Repeat for some other files that I am working on concurrently. Repeat every time I reboot or switch diagramming task / context. It turns out info alot of clicks.

I am looking for a way to make this little task of opening up a set of diagrams from my device quicker and more efficient.

* **Drag and drop** - this would be great, but does not work. It does open the file you drag but Save does not work when you open a local file using drag and drop: If I drag a local file to drawio, it seems to think it is a new file and Save will prompt you to set name and choose storage option.
* **Recent** - this would be nice, but it is only populated with entries from google drive and I assume onedrive (which I dont use currently).
* **IDEA:** **Keyboard shortcut for open**, preferably open from device or a perhaps configureable open alternative, so the favorite storage option can be set for the open keyboard shortcut. I think the storage options that appear in the recent file list are not as crucial.

Since it is a local file, security is an (the) issue for the web app. I can imagine security blocks alot of options.

Any ideas or tips greatly appreciated! :-) 

Thanks for a great tool! 🙇‍♂️

BR! /marcus

**Comment by Luzie67** [comment]
I agree. Please add a Keyboard Shortcut for opening a file from a local device.

Thanks in advance.

Luzie67
